Automattic/vip-go-mu-plugins
# Upgrade Jetpack to 11.1.2

## Description
Upgrades Jetpack from 11.1.1 to 11.1.2

## Changelog Description

### Plugin Updated: Jetpack 11.1

We upgraded Jetpack 11.1.1 to 11.1.2

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
Check out PR
Use an unpinned local site
Go to Jetpack > Settings and look for 11.1.2 as the default version
